CONVENTIONS.md

```diff
@@ -470,9 +470,9 @@ for all containers in the `openshift-monitoring` namespace:
 # CPU request / usage of the SDN container
 scalar(
   max(kube_pod_container_resource_requests{namespace="openshift-sdn", container="sdn", resource="cpu"}) /
-  max(max_over_time(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate{namespace="openshift-sdn", container="sdn"}[120m]))) * 
+  max(max_over_time(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{namespace="openshift-sdn", container="sdn"}[120m]))) *
 # CPU usage of each container in the openshift-monitoring namespace
-max by (pod, container) (node_namespace_pod_container:container_cpu_usage_seconds_total:sum_rate{namespace="openshift-monitoring"})
+max by (pod, container) (node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{namespace="openshift-monitoring"})
```

Comment:
More broadly, trying to mix `openshift-monitoring` and `openshift-sdn` results doesn't make sense to me.  Perhaps this was intended to be commented out as an example of changing namespaces and dropping over-time aggregation?  I'd expect something like:

```
sort_desc(
  # Calculate the 90th percentile of CPU usage over the past hour and add 10% to that 
  1.1 * (max by (pod, container) (
    quantile_over_time(0.9, node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{namespace=~"openshift-.*", container != "POD", container!=""}[60m]))
  ) /
  # Calculate the maximum requested CPU per pod and container
  max by (pod, container) (kube_pod_container_resource_requests{namespace=~"openshift-.*", resource="cpu", container!="", container!="POD"}) 
)
```

Or, if folks don't want to weight for bursts, dropping to `avg_over_time`:

```
sort_desc(
  # Calculate the average CPU usage over the past 30 minutes
  (avg by (pod, container) (
    avg_over_time(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{namespace=~"openshift-.*", container != "POD", container!=""}[30m]))
  ) /
  # Calculate the maximum requested CPU per pod and container
  max by (pod, container) (kube_pod_container_resource_requests{namespace=~"openshift-.*", resource="cpu", container!="", container!="POD"}) 
)
```